components/Layout.jsx

```diff
@@ -17,6 +28,7 @@ function Layout({ children }) {
   const toggleDarkMode = () => {
     document.body.classList.toggle('dark');
     document.body.classList.toggle('bg-gray-800');
```

Comment:
Toggle this class on the `html` tag & give the `body` tag the class of `dark:bg-gray-800 bg-white`.